src/components/NewCourse.js

```diff
@@ -104,6 +109,44 @@ class NewCourse extends React.Component {
             </FormText>
           </Col>
         </FormGroup>
+        <FormGroup row>
+          <Label for="name" sm={3}>
+            Unlisted Course
```

Comment:
This label can just be "Unlisted", it's clear from the context that this is for a course.